mobile/performance.html

```diff
@@ -65,6 +65,10 @@
           <p><a data-featureid="server-timing">Server Timing</a> enables a server to communicate performance metrics about the request-response cycle to the user agent, and allows applications to act on these metrics to optimize application delivery.</p>
           <p>The <a data-featureid="longtasks">Long Tasks API</a> exposes a mechanism to detect long running tasks that monopolize the user interface's main thread for extended periods of time.</p>
         </div>
+
+        <div data-feature="Low-level Bytecode Format">
+            <p><a data-featureid="webassembly">WebAssembly</a> is a low-level bytecode format that runs with near-native speed in web browsers, and supports compilation from C, C++, and other languages.</p>
```

Comment:
That's all true but seems to suggest that JavaScript is out of the picture, here. I suggest to steal some text from the [WebAssembly WG charter](http://www.w3.org/2017/08/wasm-charter)

```html
<p><a data-featureid="webassembly">WebAssembly</a> is a low-level bytecode format that runs with near-native speed in web browsers and supports compilation from C, C++, and other languages. It also defines an execution environment that attempts to maximize performance and interoperate gracefully with JavaScript and the Web, while ensuring security and consistent behavior across a variety of implementations.</p>
```